components/media-carousel/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "media-carousel",
-  "version": "2.0.1",
```

Comment:
Same with this, bumping up 2 versions is correct?

Reply:
See comment above.